Amethyst/Managers/WindowManager.swift

```diff
@@ -634,6 +656,17 @@ final class WindowManager: NSObject, MouseStateKeeperDelegate {
 
             if screenManager == nil {
                 screenManager = ScreenManager(screen: screen, screenIdentifier: screenIdentifier, delegate: self, userConfiguration: userConfiguration)
+                screenManager!.onReflowInitiation = { [unowned self] in
```

Comment:
Using `unowned` here is actually unsafe. I think right now the lifetime of the screen manager is tied to the lifetime of the window manager, but that's not a guarantee that is explicitly made. In this case I think it would be better to use `weak` as `self` is not the owner of the object in question; the screen manager owns the object (the block).